### DISCUSSION TITLE
Is there a way to 'get' the current tab index

### DISCUSSION THREAD

**Discussion by timmyd87** [excid3/tailwindcss-stimulus-components#120] [Q&A]
Hi Community!!

I know we can change tab by mentioning the index, but is there a way get the current active tab's index? Every method i've tried returns '0'. My use case is i have dynamic tabs and would like to have a 'next' & 'previous' button on the panel.

**Comment by excid3** [comment]
`index` is what's used for getting / setting the current active tab. https://github.com/excid3/tailwindcss-stimulus-components/blob/master/src/tabs.js#L54

**Reply by timmyd87** [reply]
Using 'this.index' or the getter/setter in an extended controller is only returning and index of '0' for me, if i use the specific method of 'this.tabTargets.indexOf(event.currentTarget)' i get '-1'. I would have thought extending the controller i should be able to use the same methods with super??
